node-notes-cli/main.js

```diff
@@ -0,0 +1,71 @@
+const fs = require('fs');
+
+const dataJSON = require('./data.json');
+
+const argv = process.argv
+
+let command;
+let argument;
+let argument2;
+
+if (argv.length === 5) {
+  command = argv[2];
+  argument = argv[3];
+  argument2 = argv[4]
+} else if (argv.length === 4){
+  command = argv[2]
+  argument = argv[3];
+} else {
+  command = argv[2];
+}
+
+//functions
+function read() {
+  for (let id in dataJSON.notes) {
+    console.log(`${id}: ${dataJSON.notes[id]}`);
+  }
+}
+
+function create() {
+  let nextID = dataJSON.nextId;
+  dataJSON.notes[nextID] = argument;
+  dataJSON.nextId = nextID + 1
+  const data = JSON.stringify(dataJSON, null, 2);
+  return data
+}
```

Comment:
What happens if the user forgets supply the contents for their new note?

Reply:
addressed